src/rail/estimation/algos/deepdisc.py

```diff
@@ -374,20 +380,13 @@ def run(self):
         run_name = self.config.run_name
 
         cfg = LazyConfig.load(cfgfile)
-        cfg.OUTPUT_DIR = output_dir        
-        
+        cfg.OUTPUT_DIR = output_dir
+
         #cfg.train.init_checkpoint = os.path.join(output_dir, run_name) + ".pth"
         cfg.MODEL.WEIGHTS = os.path.join(output_dir, run_name) + ".pth"
```

Comment:
I think we can probably get ride of this line now.

Reply:
We can also remove line 386 here

Reply:
Done